docs-src/cloud/tcld/feature/toggle-apikey.md

```diff
@@ -0,0 +1,27 @@
+---
```

Comment:
api key is already in public preview. So there is no `tcld feature toggle-apikey ` any more. 

I would not suggest put tcld feature toggle-X on the UI. They're usually very short lived. Otherwise, we have to keep it updated. It's a pain.